app/modules/sightings/resources.py

```diff
@@ -132,6 +132,19 @@ def _validate_assets(assets, paths_wanted):
     return matches
 
 
+def _validate_annotations(enc_json):
```

Comment:
the name validate implies that the function is just checking the annotations, this isn't the case, it actually returns them. 
_get_annotations?

Reply:
yeah sure, seems more accurate.